watertap/examples/flowsheets/RO_with_energy_recovery/monte_carlo_sampling_RO_ERD.py

```diff
@@ -61,9 +72,13 @@ def run_parameter_sweep(results_file, seed=None, use_LHS=False):
     # Run the parameter sweep study using num_samples randomly drawn from the above range
     num_samples = 10
 
+    # Filename of the h5 file that we want to write but without the extension
+    h5_fname = "monte_carlo_results"
+
     # Run the parameter sweep
-    global_results = parameter_sweep(m, sweep_params, outputs, results_file=results_file, 
-        optimize_function=optimize, optimize_kwargs={'solver':solver}, num_samples=num_samples, seed=seed)
+    global_results = parameter_sweep(m, sweep_params, outputs, csv_results_file=results_file,
+        results_fname=h5_fname, optimize_function=optimize, optimize_kwargs={'solver':solver},
```

Comment:
Why is the name of the csv file `csv_results_file` and the h5 file `results_fname`. Is there a good reason to output both? Could we just have `results_file_name` and write *either* a csv *or* an h5 file? We could have an additional flag for it and/or infer it from the file extension.

Reply:
The ability to write csv files will go away in a later PR since the dictionary stored within the h5 file is much more exhaustive and makes the csv file redundant. The reason the two exist concurrently is because I wanted to make sure that the transition from one to another happens smoothly.

Reply:
I personally think we should maintain both capabilities. While h5 files have their uses, csv files have the advantage of being human-readable, can be opened in Excel, and are easily parsed. Further, you may not *want* to save all the data from a model -- for doing large sweeps this could become an issue.